electrumsv/regtest_support.py

```diff
@@ -0,0 +1,114 @@
+import json
+import logging
+import os
+from typing import List
+
+import requests
+from bitcoinx import Headers, MissingHeader, CheckPoint, bits_to_work
+from electrumsv.bitcoin import COINBASE_MATURITY
```

Comment:
alphabetical sorting of installed packages by name, then a new line, then electrumsv imports